src/common/options/rgw.yaml.in

```diff
@@ -3692,3 +3692,23 @@ options:
   default: tank
   services:
   - rgw
+- name: rgw_d4n_directory_host
+  type: str
+  level: advanced
+  desc: The rgw directory host
+  default: 127.0.0.1
+  services: 
+  - rgw
+  flags:
+  - startup
+  with_legacy: true
+- name: rgw_d4n_directory_port
```

Comment:
Have we thought about combining the d4n port and host config variables into a `rgw_d4n_directory_endpoint(s)` variable? Is there a good reason to keep them separate?

Reply:
It's more likely that the port will change than the host, so it's a bit more flexible when separated in this way. There is no other reason to keep it separate.